network/src/peers/peer/mod.rs

```diff
@@ -0,0 +1,37 @@
+// Copyright (C) 2019-2021 Aleo Systems Inc.
+// This file is part of the snarkOS library.
+
+// The snarkOS library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// The snarkOS library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with the snarkOS library. If not, see <https://www.gnu.org/licenses/>.
+
+mod cipher;
+mod connector;
+mod handshake;
+mod network;
+pub mod peer;
+mod peer_events;
+pub mod peer_quality;
+mod receiver;
+
+mod inbound_handler;
+mod outbound_handler;
+
+pub use outbound_handler::*;
+pub use peer::*;
+pub use peer_quality::*;
+// used in integration tests
+#[doc(hidden)]
+pub use cipher::Cipher;
+#[doc(hidden)]
+pub use network::*;
+pub use peer_events::*;
```

Comment:
Please cleanup this import declaration

Reply:
ill order them up i suppose, is there anything your looking for in particular here?